deploy/charts/external-secrets/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: external-secrets
 description: External secret management for Kubernetes
 type: application
-version: "0.1.0"
+version: "0.1.1"
```

Comment:
Normally bumping this would be desired, but 0.1.0 is not technically released, so not sure this is required.